grafana/alerting-loki-logs/step1.md

```diff
@@ -0,0 +1,31 @@
+# Before you begin
+
+## Grafana Cloud users
+
+As a Grafana Cloud user, you don’t have to install anything.
+
+## Grafana OSS users
+
+In order to run a Grafana stack locally, ensure you have the following applications installed.
+
+- [Docker Compose](https://docs.docker.com/get-docker/) (included in Docker for Desktop for macOS and Windows)
+
+- [Git](https://git-scm.com/)
+
+To demonstrate the observation of data using the Grafana stack, download the files to your local machine.
+
+1. Download and save a Docker compose file to run Grafana, Loki and Promtail.
+
+```bash
+wget https://raw.githubusercontent.com/grafana/loki/v2.8.0/production/docker-compose.yaml -O docker-compose.yaml
+```{{exec}}
+
+1. Run the Grafana stack.
+
+```bash
+docker compose up -d
```

Comment:
This doesn't work since killercoda uses `docker-compose`

What you can do is just add this to the metadata at the top
```
  preprocessing:
    substitutions:
      - regexp: docker compose
        replacement: docker-compose
```